src/components/CocktailItem/CocktailItem.jsx

```diff
@@ -8,32 +11,52 @@ import {
   CocktailCard,
   CocktailLabel,
   CocktailTitle,
-  MoreButton,
-  RemoveButton,
-  // Icon,
-} from "./CocktailItem.styled";
+  MoreLink,
+} from './CocktailItem.styled';
+import RemoveButton from '../RemoveButton/RemoveButton';
 
-const CocktailItem = ({ cocktail }) => {
-  const { drink, alcoholic, description, drinkThumb } = cocktail;
+const CocktailItem = ({ cocktail, page }) => {
+  const { id, drink, alcoholic, description, drinkThumb } = cocktail;
```

Comment:
```suggestion
  const { _id, drink, alcoholic, description, drinkThumb } = cocktail;
```
замінити всі id на _id